webrender/src/prim_store.rs

```diff
@@ -109,7 +109,11 @@ pub enum PrimitiveCacheKey {
 #[derive(Debug)]
 pub struct PrimitiveMetadata {
     pub is_opaque: bool,
-    pub clip_source: Box<ClipSource>,
+    // TODO(gw): It's only a very small subset of primitives
+    // that end up with > 1 clip region. Consider an optimization
+    // for this case - perhaps an enum that either holds one
+    // clip or a Vec of clips?
```

Comment:
I don't think the optimization is necessary, given that we are using `Box<ClipSource>` already.